woocommerce/woocommerce-android
# Unified Login: No WPcom found error screen

Closes #2815. During the WordPress.com login flow, if the user enters an email address that is not associated with a WPcom account, then this new error screen is shown:

Before | After (light) | After (dark)
-- | -- | --
![Screenshot_1601505407_framed](https://user-images.githubusercontent.com/5810477/94746995-f7d08300-034b-11eb-911d-01e231d95c23.png)|![Screenshot_1601580317_framed](https://user-images.githubusercontent.com/5810477/94854246-9ca99980-03fa-11eb-8174-7bf6cf40d4f8.png)|![Screenshot_1601580341_framed](https://user-images.githubusercontent.com/5810477/94854254-a0d5b700-03fa-11eb-8518-daf14658eabc.png)




### Tracks events
The following `Flow`, `Step` and `Click` tracks events were added:
- Step: `no_wpcom_account_found` - tracked as soon as this screen is shown
- Click: `help_finding_connected_email` - when clicking on the link to get help finding connected email
- Click: `try_another_account` - when clicking try another account
- Click: `login_with_site_address` - when clicking the option to login with site address

Track event samples:
```
Tracked: unified_login_step, Properties: {"source":"default","flow":"google_login","step":"no_wpcom_account_found"}
Tracked: unified_login_interaction, Properties: {"source":"default","flow":"google_login","step":"no_wpcom_account_found","click":"help_finding_connected_email"}
Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"no_wpcom_account_found","click":"try_another_account"}
Tracked: unified_login_interaction, Properties: {"source":"default","flow":"google_login","step":"no_wpcom_account_found","click":"login_with_site_address"}
```

### To Test
1. Launch the app to login
2. Click on **Continue with WordPress.com**
3. Enter an email address not associated with a WordPress.com account - verify new error screen is displayed
4. Click each button on screen to verify action and track event.


Update release notes:
